docs/Hooks.md

```diff
@@ -26,18 +26,31 @@ fastify.addHook('onResponse', (res, next) => {
 })
 ```
 
+| Parameter   |  Description  |
+|-------------|:-------------:|
+| req |  Node.js [IncomingMessage](https://nodejs.org/api/http.html#http_class_http_incomingmessage) |
+| reply | Fastify [Reply](https://github.com/fastify/fastify/blob/master/docs/Reply.md) interface |
+| res | Node.js [ServerResponse](https://nodejs.org/api/http.html#http_class_http_serverresponse) |
```

Comment:
I think this should be moved up into the position currently occupied by `reply`.